app/index.html

```diff
@@ -13,10 +13,10 @@
 
     <!-- iOS home screen icons -->
     <meta name="apple-mobile-web-app-title" content="react boilerplate">
-    <link rel="apple-touch-icon" sizes="152x152" href="icon-120x120.png">
+    <link rel="apple-touch-icon" sizes="120x120" href="icon-120x120.png">
     <link rel="apple-touch-icon" sizes="152x152" href="icon-152x152.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="icon-167x167.png">
-    <link rel="apple-touch-icon" sizes="167x167" href="icon-180x180.png">
+    <link rel="apple-touch-icon" sizes="167x167" href="icon-167x167.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="icon-180x180.png">
```

Comment:
@tomazy Absolutely 😄, puns aside I agree, just puhed up the changes. Out of interest I tested on iPhone 5s (iOS 9.3.1) and it appears to work with or without relative on both pages so must be some Apple magic going on there.